CMakeLists.txt

```diff
@@ -473,6 +476,10 @@ find_feature(USE_SQLITE3 "sqlite3")
 find_feature(USE_ELF "libelf")
 find_feature(ENABLE_PYTHON "PythonLibs")
 
+if(USE_PLEDGE)
```

Comment:
This should go above the `find_feature` calls. Probably above the block of `set(WANT_`, but not necessarily.